Detectors/Upgrade/ITS3/macros/test/run_sim_its3.C

```diff
@@ -0,0 +1,194 @@
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+#include <Rtypes.h>
+#include <TSystem.h>
+#include <TMath.h>
```

Comment:
is this file really needed? If possible, consider directly using o2-sim.

Reply:
No, it is a leftover, I'll move it away.